src/prefect/storage/_healthcheck.py

```diff
@@ -35,9 +37,10 @@ def system_check(python_version: str):
 def cloudpickle_deserialization_check(flow_file_paths: list):
     flows = []
     for flow_file in flow_file_paths:
-        with open(flow_file, "rb") as f:
+        with open(flow_file, encoding="utf-8") as f:
             try:
-                flows.append(cloudpickle.load(f))
+                flow_bytes = binascii.a2b_base64(json.loads(f.read())["flow"])
```

Comment:
A comment that this is matching the implementation of `flow_from_bytes_pickle` would be nice